<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskListPrefWidget</class>
 <widget class="QWidget" name="TaskListPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task List</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="mTaskTagCustomizeLayout">
     <item row="5" column="1">
      <widget class="QPushButton" name="mRemoveButton">
       <property name="toolTip">
        <string>Click to remove the selected task tag from the list.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="remove">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="mTaskTagLineEdit">
       <property name="toolTip">
        <string>Type a new task tag to add to the list.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLabel" name="mNewTagLabel">
       <property name="text">
        <string>Add New Tag:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="mAddButton">
       <property name="toolTip">
        <string>Click to add the task tag provided in the text field.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="add">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QListWidget" name="mTaskTagList">
       <property name="toolTip">
        <string>List of available task tags used by the Task List.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="mTaskTagLabel">
       <property name="text">
        <string>Task Tags:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="mColumnCustomizeLayout">
     <item row="2" column="1">
      <widget class="QPushButton" name="mDown">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="down">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="mAvailableColumnsLabel">
       <property name="text">
        <string>Available Columns:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="mVisibleColumnsLabel">
       <property name="text">
        <string>Visible Columns:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="mLeft">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="left">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="mUp">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="up">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="mRight">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="right">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="4">
      <widget class="QListWidget" name="mVisibleColumns"/>
     </item>
     <item row="1" column="0" rowspan="4">
      <widget class="QListWidget" name="mAvailableColumns"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="mPathFormatLayout">
     <item row="0" column="1">
      <widget class="QRadioButton" name="mAbsolutePathFormat">
       <property name="text">
        <string>Absolute</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="mRelativePathFormat">
       <property name="text">
        <string>Relative to Scenario</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="mPathFormatLabel">
       <property name="text">
        <string>Path Format:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="mSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
